Exercise_C/index.css

```diff
@@ -2,3 +2,34 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.box
+{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    height: 200px;
+}
+
+#blue
+{
+    background-color:blue;
+    width: 100px;
+    height: 100px;
+
+}
+
+#green
+{
+    background-color:green;
+    width: 100px;
+    height: 100px;
+}
+
+#red
+{
+    background-color:red;
+    width: 100px;
+    height: 100px;
+    align-self: flex-end;
+
+}
```

Comment:
That's an excellent use of flex. This shows that you have a decent understanding of how flex works